purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -473,6 +474,42 @@ class PurchasesTest {
         assertThat(callCount).isEqualTo(1)
     }
 
+    @Test
+    fun `when making a deferred upgrade, completion is called with null purchase for new version`() {
```

Comment:
I think this test name is outdated and it should say the completion has the store transaction?

Reply:
Renamed.